src/wingetui/Assets/Utilities/reset_winget_sources.cmd

```diff
@@ -8,7 +8,9 @@ echo This script will reset winget sources.
 echo . Please wait... 
 echo winget path: %wingetpath%
 echo Deleting Winget local data sources...
+powershell Set-ExecutionPolicy Unrestricted
 powershell -file "%~dp0\delete_winget_databases.ps1"
+powershell Set-ExecutionPolicy Default
```

Comment:
Rather than changing the execution policy in a separate command, could you instead set it just for the specific PowerShell script?

```suggestion
powershell -File "%~dp0\delete_winget_databases.ps1" -ExecutionPolicy Bypass
```

You can quickly test that this works in cmd by running
``` batch
powershell Get-ExecutionPolicy
powershell -ExecutionPolicy Bypass -Command Get-ExecutionPolicy
powershell Get-ExecutionPolicy
```

[PowerShell CLI Reference](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_powershell_exe?view=powershell-5.1#-executionpolicy-executionpolicy)